content/10.diagnostics.md

```diff
@@ -131,11 +130,12 @@ First, the authors evaluated samples from the 103 suspected cases.
 Using RT-qPCR, 29 (28%) were identified as positive, 25 (24%) as negative, and 49 (48%) as borderline, i.e., the C~t~ value was higher than the positive threshold of 35 but lower than the negative threshold of 40.
 When the 61 negative and borderline samples were reanalyzed with ddPCR, 19 (31%) of the negative and 42 (69%) of the borderline samples were identified as positive.
 All of the suspected cases were later confirmed to be COVID-19 through a combination of symptom development and RT-qPCR resampling, indicating that ddPCR improved the overall detection rate compared to RT-qPCR from 28.2% to 87.4%.
+
 They repeated this analysis in patient samples from contacts and close contacts.
 Patients who tested negative with both methods (n = 48) were observed to remain healthy over a period of 14 days.
-Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline. 
+Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline.
 All of the samples that tested positive using RT-qPCR also tested positive using ddPCR.
-In contrast, the negative result and all but one of the borderline results were identified as positive by RT-ddPCR, and these patients were later determined to be SARS-CoV-2 positive based on clinical evaluation and repeated molecular sampling. <!--It is not entirely clear to me from the methods & supplement whether this resampling was done with RT-qPCR (section 3.5 in the reference). However, as noted above, they do specify the technique used for suspected cases (right about Fig 3 in the paper).-->
```

Comment:
Due to the lack of info in this paper, it seems inadvisable to clarify this statement, so I'm removing the comments.